### PR TITLE
#165596325 implement seeding database and pagination

### PR DESCRIPTION
**Pull request by devPinheiro**
#### What does this PR do?
- It contains a feature for seeding database and pagination

#### Description of Task to be completed?
- create seeder files
- update package.json to seed database
- write test for authors and books endpoints pagination
- implement pagination within authors and books controller

#### How should this be manually tested?
- After cloning the repo, cd into the root folder

- Install all necessary dependencies by running 

  ``npm install``

- Launch the node server with

  ``npm run dev``

- To run local tests using jest
  
  ``npm run test``

#### Any background context you want to provide?
- You should have Nodejs installed

#### What are the relevant pivotal tracker stories?
- PT Story link - `[#165596325] (https://www.pivotaltracker.com/story/show/165596325)`

### CODE REVIEW

File: __tests__/author.spec.js [hunk 1]
```diff
@@ -41,6 +45,70 @@ describe('AUTHOR API ENDPOINTS', () => {
...
+            const { status, body } = await server().get(
+                '/api/v1/authors?limit=e'
+            );
+
+            expect(status).toBe(422);
+            expect(body).toMatchSnapshot();
+        });
+
+        it('should get all authors', async () => {
+            const firstAuthor = getAuthor();
```
Comment (bahdcoder) [inline __tests__/author.spec.js]: Please use a loop to generate this.

File: __tests__/books.spec.js [hunk 2]
```diff
@@ -102,18 +102,61 @@ describe('GET ALL BOOKS API ENDPOINT', () => {
...
+        expect(status).toBe(422);
+        expect(body).toMatchSnapshot();
+    });
+
     it('should return all books', async () => {
         const firstBook = getBook();
         firstBook.isbn = '128b4v389028074';
         const secondBook = getBook();
         secondBook.isbn = '9204798753002380';
+        const thirdBook = getBook();
```
Comment (bahdcoder) [inline __tests__/books.spec.js]: Please use a loop to generate this. A better way is to create 2 items, and pass `limit` to be 1 so we get two pages

File: server/config/database.js [hunk 3]
```diff
@@ -9,6 +9,10 @@ export default () => ({
         migrations: {
             tableName: 'knex_migrations',
             directory: './server/migrations'
+        },
+        seeds: {
```
Comment (bahdcoder) [inline server/config/database.js]: Why do we have this here ? We are not using knex seeds in the traditional sense

Comment (devPinheiro) [reply server/config/database.js]: feedback implemented

File: __tests__/books.spec.js (hunk 2 above)
Comment (devPinheiro) [reply __tests__/books.spec.js]: feedback implemented

File: __tests__/author.spec.js (hunk 1 above)
Comment (devPinheiro) [reply __tests__/author.spec.js]: feedback implemented

**Pull request merged**